lms/djangoapps/grades/signals/handlers.py

```diff
@@ -120,7 +126,7 @@ def score_published_handler(sender, block, user, raw_earned, raw_possible, only_
             user_id=user.id,
             course_id=unicode(block.location.course_key),
             usage_id=unicode(block.location),
-            only_if_higher=only_if_higher,
+            only_if_higher=only_if_higher
```

Comment:
Also missed undoing this after a previous implementation. Thanks for catching it.